webapp/wmservices/pom.xml

```diff
@@ -9,82 +9,96 @@
 	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES 
 	OR CONDITIONS OF ANY KIND, either express or implied. See the License for 
 	the specific language governing permissions and limitations under the License. -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.apache.oodt</groupId>
-		<artifactId>oodt-core</artifactId>
-		<version>1.9-SNAPSHOT</version>
-		<relativePath>../../core/pom.xml</relativePath>
-	</parent>
-	<artifactId>workflow-services</artifactId>
-	<packaging>war</packaging>
-	<name>CAS Workflow REST Services</name>
-	<description>
-          A set of REST-ful services for the Workflow Manager.
-    </description>
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-jar-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>make-a-jar</id>
-						<phase>compile</phase>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-install-plugin</artifactId>
-				<executions>
-					<execution>
-						<phase>install</phase>
-						<goals>
-							<goal>install-file</goal>
-						</goals>
-						<configuration>
-							<packaging>jar</packaging>
-							<artifactId>${project.artifactId}</artifactId>
-							<groupId>${project.groupId}</groupId>
-							<version>${project.version}</version>
-							<file>
-								${project.build.directory}/${project.artifactId}-${project.version}.jar
-							</file>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-		</plugins>
-	</build>
-	<dependencies>
-		<dependency>
-			<groupId>org.apache.oodt</groupId>
-			<artifactId>cas-workflow</artifactId>
-			<scope>compile</scope>
-		</dependency>
-		<dependency>
-			<groupId>javax.servlet</groupId>
-			<artifactId>servlet-api</artifactId>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.cxf</groupId>
-			<artifactId>cxf-rt-frontend-jaxrs</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.cxf</groupId>
-			<artifactId>cxf-rt-transports-local</artifactId>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
-</project>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Just an inquiry. What's the difference between LF and CLRF. Both are line endings, right?

Reply:
CRLF (Carriage return line feed \r\n) is used by windows operating systems. LF (\n) is used by unix mostly. Windows will usually replace LF with CRLF when cloning repositories. This will cause git to consider the whole file as changed. In this file, I noticed that line endings have changed from LF to CRLF in your version. In future, if we want to see who changed a specific line, it will be you who will get annotated due to this, not the actual author.